mobile/native/src/ln_dlc/mod.rs

```diff
@@ -369,10 +369,14 @@ pub fn run(seed_dir: String, runtime: &Runtime) -> Result<()> {
         // TODO: This might not be necessary once we rewrite the on-chain wallet with bdk:1.0.0.
```

Comment:
We will probably not need it for the on-chain sync. But the off-chain balance will definitely fail in regards to the linked issue, since the contract was not backed up breaking the restored app.